hera_pspec/plot.py

```diff
@@ -1076,4 +1077,4 @@ def _round_sigfig(x, up=True):
     if up:
         return np.ceil(10**sigfigs * x) / 10**sigfigs
     else:
-        return np.floor(10**sigfigs * x) / 10**sigfigs
\ No newline at end of file
+        return np.floor(10**sigfigs * x) / 10**sigfigs
```

Comment:
Line not tested according to codecov

Reply:
I don't think this file was covered before?